sql/dql/basic-queries/where-2.md

```diff
@@ -31,65 +31,91 @@ aspects:
 ---
 ## Content
 
-### `AND`, `OR`, `NOT`, `BETWEEN`
+### `AND`, `OR`, `NOT`
 
-The predicates we use in `WHERE` can be conjoined using logical operators like `AND`, `OR` and `NOT`. They work as you would expect them to; you can include only the rows which satisfy both or either of two conditions using `AND` and `OR`, respectively. `NOT` negates the truth value of the predicate, so it now selects the rows that don't satisfy the condition.
+As you have seen in the previous insight, the predicates we use in `WHERE` can be conjoined using logical operators like `AND`, `OR` and `NOT`. These operators are used in combination with filtering conditions (e.g. `salary > 1000`) and can be used for:
 
-```sql
-SELECT *
-FROM pokemon
-WHERE name = 'charmander'
-  OR name = 'metapod';
-```
+- returning records for which two (or more) conditions are true at the same time with `AND`
+- returning records for which any of two (or more) conditions are true with `OR`
+- negating a condition with `NOT`
 
-This shows the rows where the name is an exact match of either 'charmander' or 'metapod'.
+They work as you would expect them to; you can include only the rows which satisfy both or either of two conditions using `AND` and `OR`, respectively. `NOT` negates the truth value of the predicate, so it now selects the rows that don't satisfy the condition.
+
+As always, let's take a look at this `pokemon` table:
```

Comment:
We might want to remove the "As always" part since we don't want insight location to be coupled with its location when it doesn't have to be.